include/aws/testing/compression/huffman.inl

```diff
@@ -131,6 +137,10 @@ int huffman_test_transitive_chunked(
         *error_string = "too much data encoded";
         return AWS_OP_ERR;
     }
+    if (encoded_size && intermediate_buf.len != encoded_size) {
+        *error_string = "encoded length is incorrect";
+        return AWS_OP_ERR;
```

Comment:
this branch, and maybe some others, don't aws_raise_error()

Reply:
This is the test harness, it just sets the error string and returns error.